cmd/keeper/cmd/keeper.go

```diff
@@ -779,10 +781,53 @@ func (p *PostgresKeeper) Start(ctx context.Context) {
 
 		case <-endUpdateKeeperInfo:
 			updateKeeperInfoTimerCh = time.NewTimer(p.sleepInterval).C
+		case <-restartPGTimerCh:
+			go func() {
+				p.restartPostgresIfScheduled(ctx)
```

Comment:
I will prefer to not add a new time and a new function the read clusterdata again, this will also cause some race condition when an user asks to restart postgres while the main loop is acting on it. So I'll do this in the main check.